src/DotFlyer.Service/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -50,6 +50,8 @@ public static IServiceCollection WithDependencies(this IServiceCollection servic
                     .CreateProcessor(configuration["AzureServiceBus:TopicNameForSMS"], configuration["AzureServiceBus:SubscriptionName"]))
                     .WithName(SMSTopicProcessor.ProcessorName);
 
+            clientBuilder.AddBlobServiceClient(new Uri($"https://{configuration["StorageAccount:Name"]}.blob.core.windows.net"));
```

Comment:
I wonder if we should do some validations... e.g. check that the resulting string is correct url etc...
that the string for the container is valid etc, (length, characters etc.)

Reply:
this one may be not a great example then... I guess consider it for anything human controlled?